.travis/install.sh

```diff
@@ -0,0 +1,78 @@
+#!/bin/bash
+
+# This script was adapted from work by Keith James (keithj). The original source
+# can be found as part of the wtsi-npg/data_handling project here:
+#
+#   https://github.com/wtsi-npg/data_handling
+
+set -e -x
+
+sudo apt-get install libgd2-xpm-dev # For npg_tracking
+sudo apt-get install liblzma-dev # For npg_qc
+sudo apt-get install --yes nodejs
+
+# CPAN
+cpanm --quiet --notest Alien::Tidyp # For npg_tracking
+cpanm --quiet --notest Module::Build
+cpanm --quiet --notest Config::Auto # For npg_qc
+cpanm --quiet --notest Readonly
```

Comment:
As I mentioned elsewhere, you are building in the wrong order, starting from npg_qc. The stack trace clearly shows that this failure is caused by a failure to compile /tmp/npg_tracking.git/lib/npg_tracking/util/config.pm (see line 508 of the travis output you refer to).